src/main/java/com/timelogr/enterprise/TimeLogrController.java

```diff
@@ -5,51 +5,55 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-//ALL WEB FILES GO IN THE resources/templates DIRECTORY
+/**
+ * ALL WEB FILES GO IN THE resources/templates DIRECTORY
+ */
 @Controller
 @RequestMapping("/")
 public class TimeLogrController {
-    private TimeLogrServices timeLogrServices;
 
+    private final TimeLogrServices timeLogrServices;
```

Comment:
Changed the timeLogrServices variable to a constant (this link may help): https://stackoverflow.com/questions/7563652/static-final-variable-in-java